File: saude_java/app/src/main/java/saude_java/IMC.java
package saude_java;

public class IMC {
    
    public static double calcular(double peso, double altura) {
        if (altura <= 0) {
            throw new IllegalArgumentException("Altura deve ser maior que zero");
        }
        double imc = peso / (altura * altura);
        return Math.round(imc * 100.0) / 100.0;
    }
    
    public static String classificacao(double imc) {
        if (imc < 18.5) {
            return "Abaixo do peso";
        } else if (imc < 25) {
            return "Peso normal";
        } else if (imc < 30) {
            return "Sobrepeso";
        } else if (imc < 35) {
            return "Obesidade Grau 1";
        } else if (imc < 40) {
            return "Obesidade Grau 2";
        } else {
            return "Obesidade Grau 3 (mórbida)";
        }
    }
}

File: saude_python/src/saude_python/imc.py
class IMC:
    @staticmethod
    def calcular(peso, altura):
        if altura <= 0:
            raise ValueError("Altura deve ser maior que zero")
        imc = peso / (altura ** 2)
        return round(imc, 2)

    @staticmethod
    def classificacao(imc):
        if imc < 18.5:
            return "Abaixo do peso"
        elif 18.5 <= imc <= 24.9:
            return "Peso normal"
        elif 25 <= imc <= 29.9:
            return "Sobrepeso"
        elif 30 <= imc <= 34.9:
            return "Obesidade Grau 1"
        elif 35 <= imc <= 39.9:
            return "Obesidade Grau 2"
        else:
            return "Obesidade Grau 3 (mórbida)"
